src/SFA.DAS.QnA.Application/Commands/SetPageAnswers/SetAnswersBase.cs

```diff
@@ -55,7 +55,7 @@ protected void UpdateApplicationData(string pageId, List<Answer> submittedAnswer
         {
             if (application != null)
             {
-                var applicationData = JObject.Parse(application.ApplicationData ?? "{}");
+                var applicationData = JsonObject.Parse(application.ApplicationData ?? "{}");
```

Comment:
This returns a `JsonNode` (not a `JsonObject`), but since we always cast it to a `JsonObject` before using it, might as well do a one-time cast here instead and remove the two casts down below

```suggestion
                var applicationData = JsonNode.Parse(application.ApplicationData ?? "{}").AsObject();
```

Reply:
done